src/infra/HttpRequester.ts

```diff
@@ -1,42 +1,18 @@
 import { HttpRequesterService } from '../services/HttpRequesterService';
-import { BeaconPayload } from '../models/BeaconPayload';
+import { AnalyticsPayload } from '../models/AnalyticsPayload';
 import fetch from 'cross-fetch';
 
 /**
  * Responsible for making web requests.
  */
 export class HttpRequester implements HttpRequesterService {
-  beacon(url: string, body: BeaconPayload): boolean {
+  post(url: string, body: AnalyticsPayload): Promise<Response> {
     const data = JSON.stringify(body);
 
-    const isNode = typeof(window) === 'undefined';
-    if (isNode) {
-      fetch(url, { method: 'POST', body: data });
-      return true;
-    }
-
-    const isBeaconSupported = !!window.navigator?.sendBeacon;
-    if (isBeaconSupported) {
-      return window.navigator.sendBeacon(url, data);
-    }
-
-    return this._beaconPolyfill(url, data);
-  }
-
-  // Navigator.sendBeacon polyfill
-  // Combination of the compact Financial Times polyfill:
-  // https://github.com/Financial-Times/
-  // polyfill-library/blob/master/polyfills/navigator/sendBeacon/polyfill.js
-  // with the async-by-default behavior of Miguel Mota's polyfill:
-  // https://github.com/miguelmota/Navigator.sendBeacon/blob/master/sendbeacon.js
-  _beaconPolyfill(url: string, data: string): boolean {
-    const event = window.event?.type;
-    const sync = event === 'unload' || event === 'beforeunload';
-    const xhr = ('XMLHttpRequest' in window) ? new XMLHttpRequest() : new ActiveXObject('Microsoft.XMLHTTP');
-    xhr.open('POST', url, !sync);
-    xhr.setRequestHeader('Accept', '*/*');
-    xhr.setRequestHeader('Content-Type', 'text/plain;charset=UTF-8');
-    xhr.send(data);
-    return true;
+    return fetch(url, {
```

Comment:
fetch doesn't work for IE, do we need some sort of polyfill?

Reply:
I don't think we need to do any polyfilling here, answers-core also doesn't do any polyfilling in core but works fine in ie11. This is probably due to cross-fetch already using whatwg-fetch under the hood.
I added `import 'core-js/stable'` to the top of the test-site's index.ts and the site works fine in ie11.

Reply:
I updated HttpRequester to use window.fetch if available the same way core does it, and retested in ie11!